src/topic/TopicMessageQuery.js

```diff
@@ -401,11 +408,25 @@ export default class TopicMessageQuery {
                         this._attempt < this._maxAttempts &&
                         this._retryHandler(error)
                     ) {
+                        const delay = Math.min(
+                            250 * 2 ** this._attempt,
+                            this._maxBackoff
+                        );
+                        console.log(
```

Comment:
This should probably be a `console.error()`. If you were using a logging framework it would be a warn level.

Reply:
That sounds like it should be `console.warn()` then; on another note we should really update to use a proper logging library.